src/services/programs.js

```diff
@@ -1107,8 +1025,290 @@ module.exports = class ProgramsHelper {
 			})
 		}
 	}
+	/**
+	 * Submit the program for review
+	 * @method
+	 * @name publish
+	 * @param {string} programId - The ID of the program for submitting for review.
+	 * @returns {JSON} - Response status of the submission
+	 */
+	static async publish(programId, userDetails) {
```

Comment:
@adithyadinesh0412, instead of passing **userDetails**, please send **userId** and **orgId** as separate parameters. Otherwise, if this function is called elsewhere, we would need to reconstruct **userDetails**. Passing parameters separately improves readability and makes the function more manageable.